index.html

```diff
@@ -552,7 +552,42 @@ <h2>Creators</h2>
         <div class="container">
           <hr size=3px width=100% color="#3D4250">
           <h2>Join us</h2>
-          <p>If you have any questions or would like to become a part of our community, please <a href="https://github.com/move-fast-and-break-things/movefastandbreakthings.club/issues" target="_blank">follow the link</a> to open a new issue. Kindly describe why you are interested in joining our community.</p>
+          <div class="TextBlock"><p>Hi!</p><br>
+
+            Welcome to Move Fast and Break Things: a community where passionate engineers collaborate, experiment, and grow together – fast. <br>
```

Comment:
Возможно отступы были не идеальными потому что этот блок, в отличии, от других не заключён в тег `p`

```suggestion
            <p>Welcome to Move Fast and Break Things: a community where passionate engineers collaborate, experiment, and grow together – fast.<p> <br>
```